test/newPet.test.js

```diff
@@ -1,31 +1,147 @@
-const { Pet, aPet } = require("../src");
+const { Pet } = require("../src");
 
 describe("newPet", () => {
     it("creates a new pet variable", () => {
-        const newPet = new Pet('Fido')        
-        expect(newPet.name).toBe('Fido')
+        const newPet = new Pet('Boris')        
+        expect(newPet.name).toBe('Boris')
     });
 });
 
 describe('constructor', () => {
   it('returns an object', () => {
-    expect(new aPet('Fido')).toBeInstanceOf(Object);
+    expect(new Pet('Boris')).toBeInstanceOf(Object);
   });
 });
 
+let boris;
+beforeEach(() => {
+  boris = new Pet('Boris')
+});
+
+
 describe("growUp", () => {
   it("increases the pet age by 1", () => {
-    let boris = new Pet('Boris')
-    boris.growUp();
-       expect(boris.age).toBe(1)
+      boris.growUp();
+      expect(boris.age).toBe(1)
   });
 });
 
 describe("growUp", () => {
   it("increases hunger by 5 & decreases fitness by 3", () => {
-    let boris = new Pet('Boris')
     boris.growUp();
        expect(boris.hunger).toBe(5)
        expect(boris.fitness).toBe(7)
   });
+});
+
+describe("walk", () => {
+  it("increases fitness by 4 but should never reach more than 10", () => {
+    boris.fitness = 8;
+    boris.walk();
+       expect(boris.fitness).toBe(10)
+       
+  });
+});
+
+describe("walk", () => {
+  it("increases fitness by 4 but should never reach more than 10", () => {
```

Comment:
I would change the description of the test, since it's the same as above, which, without reading the code, can be quite confusing. Another way of doing is creating a 2nd Pet inside of your beforeEach statement, and use it as a 2nd object to test different properties and reduce the number of tests.